faq.adoc

```diff
@@ -28,6 +28,15 @@ So far Kaitai Struct focuses on reading (parsing) only. There are plans to suppo
 
 == How does it compare to ...
 
+=== ... Python library Construct?
+
+Actually, Construct is the closest analog to Kaitai Struct. It is also a symmetric binary parsing library, but the difference is mostly that:
+
+* Construct does building and parsing while Kaitai (currently) only does parsing
```

Comment:
`building` is called [`serialization`](https://github.com/kaitai-io/kaitai_struct/issues/27) in KS.
>currently

should be a link to the issue

Reply:
Done.